gcampus/settings/prod.py

```diff
@@ -69,11 +70,33 @@
 if get_env_read_file("GCAMPUS_EMAIL_HOST_PASSWORD", None) is not None:
     EMAIL_HOST_PASSWORD = get_env_read_file("GCAMPUS_EMAIL_HOST_PASSWORD")
 DEFAULT_FROM_EMAIL = get_env_read_file(
-    "EMAIL_HOST_PASSWORD", "noreply@gewaessercampus.de"
+    "GCAMPUS_FROM_EMAIL", "GewässerCampus <noreply@gewaessercampus.de>"
```

Comment:
no direct feedback to your code but a general ToDo: Should we set up forwarding of this address to our mailbox in case somebody does respond to it? We also should provide one central mail address to get into contact (as we encourage user to contact us in some of the mails, but do not provide a mail address to do so. We may want to set this (new) mail address as a reply-to header in the mails (if possible)?